recipe/doc/recipe_guide.md

```diff
@@ -0,0 +1,87 @@
+
+requirements:
+  run:
+    # Since we’ve ignored the run export, pin manually, but set the min to just “x” since we support any libcublas within the same major release, including older versions
+    - pin_compatible(libcublas, min_pin=”x”, max_pin=”x”)
+```
+
+
+## Cross-compilation
+
+The CUDA recipes are designed to support cross-compilation.
+As such, a number of CUDA components on conda-forge are split into `noarch: generic` component packages that are named according to the supported architecture, rather than being architecture-specific packages.
+The canonical example is [the cuda-nvcc package](https://github.com/conda-forge/cuda-nvcc-feedstock/blob/main/recipe/meta.yaml) that contains the CUDA `nvcc` compiler.
+This package is split into the `cuda-nvcc` package – which is architecture specific and must be installed on the appropriate target platform (e.g.
+x86-64 Linux) – and the `cuda-nvcc-${TARGET_PLATFORM}` packages – each of which is architecture-independent and may be installed on any target, but are only suitable for use in compiling code for the specified target platform.
+This approach allows using host machines with a single platform to compile code for multiple platforms.
+
+
+## Directory structure
+
+### Linux
+
+The conda-forge CUDA packages aim to satisfy two sets of constraints.
+On one hand, the packages aim to retain as similar a structure as possible to the CUDA packages that may be installed via system package manager (e.g.
+apt).
+On the other hand, the packages aim to provide a seamless experience at both build-time and runtime within conda environments.
+To satisfy the first requirement, all files in CUDA conda packages are installed into the `$PREFIX/targets` directory.
+This includes libraries, headers, and packaging files, along with other miscellaneous files that may be present in any given package.
+To satisfy the second requirement, we symlink a number of these files into standard sysroot locations so that they can be found by standard tooling (e.g.
+CMake, compilers, ld, etc).
+Specifically, we apply the following conventions:
+Shared libraries are symlinked into `$PREFIX/lib`.
+This includes the bare name (`libcublas.so`), the SONAME, and the full name.
+Pkgconfig files are installed directly into `$PREFIX/lib/pkgconfig`.
+These are not symlinked from `$PREFIX/targets`, but are directly moved to this location.
+The reason is that pkgconfig files contain relative paths to libraries/headers/etc and the paths cannot be accurate relative to both the `targets` directory and the `lib/pkgconfig` directory.
+Since the latter is what `pkgconfig` will use, we choose to install the files into `lib/pkgconfig` and reroot the paths accordingly.
+Static libraries and header files are not symlinked into the sysroot directories.
+Instead, conda installations of `nvcc` know how to search for these packages in the correct directories.
+
+### Windows
+
+Package structure on Windows.
+Doesn’t have `x64` directory.
```

Comment:
Let's just write TODO here if it's going to be so sparse. It's okay to have incomplete docs as long as it's clear that it's incomplete and not just a dead end / lost content.

Reply:
Agreed, I'll remove.

Reply:
@vyasr By "remove" you meant removing the whole Windows section? It's gone now and it doesn't sound right to me. Windows CUDA packages are fully working.

Reply:
I haven't removed any indication that Windows is working, I've just removed the section on Windows-specific guidance for recipe maintainers because there wasn't much of note and we can fill that in later.